tools/vagrant/bootstrap.sh

```diff
@@ -2,7 +2,7 @@
 
 # Install i386 binary support on x64 system and required tools
 sudo dpkg --add-architecture i386
-echo "deb http://ppa.launchpad.net/mosquitto-dev/mosquitto-ppa/ubuntu bionic main" | tee /etc/apt/sources.list.d/mosquitto
+echo "deb http://ppa.launchpad.net/mosquitto-dev/mosquitto-ppa/ubuntu bionic main" | tee /etc/apt/sources.list.d/mosquitto.list
```

Comment:
I forgot this when I updated the docker, does replacing this line with:

add-apt-repository ppa:mosquitto-dev/mosquitto-ppa

work so this file is somewhat aligned with the dockerfile? I'm not sure what environment this script starts out with, the add-apt-repository command comes from the "software-properties-common" package.

Reply:
To be consistent with the rest of the script, I think I need
```
sudo add-apt-repository --yes ppa:mosquitto-dev/mosquitto-ppa
```
I need to make this change and check it.

Re Java 8 - no message (tested on two different machines) just the error during compile. Definitely had submodules updated.